example/ios/example/AppDelegate.m

```diff
@@ -34,31 +30,13 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   return YES;
 }
 
-// Required to register for notifications
-- (void)application:(UIApplication *)application didRegisterUserNotificationSettings:(UIUserNotificationSettings *)notificationSettings
```

Comment:
Aren't these methods still required as long as we continue to target iOS <10.0?

Also, maybe it is time to add the more recent, non-deprecated methods.